package/CMakeLists.txt

```diff
@@ -18,6 +18,8 @@ cmake_minimum_required(VERSION 3.5.0)
 # Set the project name
 project("Nebula Package")
 
+OPTION(ENABLE_PACKAGE_STORAGE "Whether package storage." ON)
```

Comment:
Why not use `ENABLE_BUILD_STORAGE` to control the package process ?

Reply:
It's not the option of this camke file.

Reply:
Could the CMakeList.txt in subdirectory not use global option ?

Reply:
They are different cmake building.This is just for package.